collections/sentinel-s2-l2a-mosaic-120.yaml

```diff
@@ -11,7 +11,7 @@ AdditionalInfoExternal:
 Image: sentinel-s2-l2a-mosaic-120/sentinel-s2-l2a-mosaic-120.png
 Resolution: 120m
 GeographicalCoverage: Land surface area between 58 degrees South and 72 degrees North.
-TemporalAvailability: 2019 (2020 coming in April 2021, past years later on)
+TemporalAvailability: 2020 (past years will be added later); old 2019 data results can be found [here](https://collections.eurodatacube.com/sentinel-s2-l2a-mosaic-120/sentinel-2-l2a-120m-mosaic-2019.html).
```

Comment:
Link here doesn't work yet. I am also wondering if there should be a yaml file kept for the 2019 Mosaic? For us this would be useful so a STAC JSON is generated and the 2019 data can also be displayed in the EDC Browser.

Reply:
> Yes the link doesnt work yet, it will work after a successful build.

Okay, great.

> about keeping 2019 yaml file , at least for https://collections.sentinel-hub.com/sentinel-s2-l2a-mosaic-120/, we had earlier on agreed with @mlubej that it is better to only have the main yaml, since 2019 mosaic will be deprecated and replaced with new version, whose details will then be added to the main yaml.

> but for edc browser case, then we need to add 2019 yaml meanwhile

For this iteration I think it's okay to deprecate this but for the next update and the future I think we should if possible not change/make yaml files disappear as this will lead to not working existing links in the EDC Browser.

Feel free to resolve this comment if you are okay with the suggestion.

Reply:
one more question,
by `not change yaml files `, do you mean the name of the file or also the content of the yaml file

 in this case for example in the future the content will be updated (e.g temporal extent,   2017,18,19 will most likely be added) but the name of the yaml  will definately stay the same.  so only the content changes.

and if any changes is made  to the contents of a yaml file, the stac json is updated with the new content  right ?

and the links only break if the name of the yaml file  is changed ?

Reply:
since the rest of the issues are resolved, let me resolve this conversation, and merge the PR,    

but you can still give me the feedback on this  later